SIRF_data_preparation/create_Hoffman_VOIs.py

```diff
@@ -47,24 +47,28 @@
     if scanID is None:
         print("Need to set the --dataset argument")
         exit(1)
-    if args["--skip_write_PETRIC_VOIs"] is not None:
+    if (args["--skip_write_PETRIC_VOIs"] is not None) and args["--skip_write_PETRIC_VOIs"]:
```

Comment:
```suggestion
    if args["--skip_write_PETRIC_VOIs"]:
```

Reply:
I guess I did some experiments with docopt to come up with this. I don't quite understand why/when docopt sets the value to False if not present, but I trust you of course.